### PR TITLE
[#2388] Enhance user project view

### LINKED ISSUE #2388: Enhance User's Project view according to the provided preview
## Changes

1. Separate services based on types
2. Change  "contact with project support" → "contact eosc experts"

## Preview

Projects sidebar services categories list v2

https://projects.invisionapp.com/share/JXOZKC8WMDU#/screens/460826934

https://projects.invisionapp.com/share/JXOZKC8WMDU#/screens/460827061_Projects_-_Details

https://projects.invisionapp.com/share/JXOZKC8WMDU#/screens/460826937

## Icon

[ico_message.svg.zip](https://github.com/cyfronet-fid/marketplace/files/7544918/ico_message.svg.zip)
)

### PR DESCRIPTION
**Pull request by wujuu**
Closes #2388 

@wujuu:
- Added separation of project items in project view, based on their order type

@jarekzet:
- Project styling overhaul

### CODE REVIEW

File: app/controllers/projects/services_controller.rb [hunk 1]
```diff
@@ -7,6 +7,9 @@ class Projects::ServicesController < ApplicationController
 
   def index
     @project_items = @project.project_items.roots
+    @order_required_items = @project_items.where(order_type: "order_required")
```
Comment (kmarszalek) [inline app/controllers/projects/services_controller.rb:10]: what about internal and external?

Comment (Marcelinna) [reply app/controllers/projects/services_controller.rb:10]: external and internal are taken into account when writing like this

Comment (kmarszalek) [reply app/controllers/projects/services_controller.rb:10]: Zrzut ekranu 2021-11-23 o 12 46 48
 it looks good. 
but there is no diff between fully_open_access and open_access.
only diff between order_required internal and external is status vs link. is it clear enough why there is status or link there? @abacz 

as I understand @Marcelinna and @bwilk will discuss this and let us know.

Comment (kmarszalek) [reply app/controllers/projects/services_controller.rb:10]: rebase :)

**Review by kmarszalek — APPROVED** [review]

File: app/controllers/projects/services_controller.rb (hunk 1 above)
Comment (wujuu) [reply app/controllers/projects/services_controller.rb:10]: Any news?

Comment (Marcelinna) [reply app/controllers/projects/services_controller.rb:10]: No, I didn't talk to @bwilk 
@wujuu @kmarszalek we leave it as it is?
or we change

Comment (wujuu) [reply app/controllers/projects/services_controller.rb:10]: I'd leave it as it. We can implement this behaviour in another ticket, if need be.

Comment (kmarszalek) [reply app/controllers/projects/services_controller.rb:10]: yep, even with the fix to the today's version :)

**Review by Marcelinna — APPROVED** [review]

**Pull request merged**